### ISSUE TITLE
Add syncId to ScatterCharts

### ISSUE THREAD

**Issue by gsawers1** [recharts/recharts#1541]
## Do you want to request a feature or report a bug?
Request a feature: add the syncId property to ScatterCharts

## What is the current behavior?
Currently adding the syncId property to two ScatterCharts with the same data but plotted differently does not sync the two charts.

## What is the expected behavior?
It would be great to be able to sync two ScatterCharts so hovering a point on one chart shows the tooltip one both charts. This provides the ability to indicate the same point on the chart when the same data is plotted two separate ways.

**Comment by infermath** [comment]
It would be also great to have scatter plot synch to other types of plots i.e scatter to linear, area etc.

**Comment by ZenHuzaini** [comment]
Hello, is there any update on this ? :)

**Comment by MichalBunkowski** [comment]
Any update on this?

**Comment by ckifer** [comment]
Not really, we're doing some refactoring currently that will make changes like this easier, easier to test, and hopefully faster to get out. Anyone is free to contribute though as always

**Comment by iamgourav** [comment]
Any update on this ??

**Comment by ckifer** [comment]
Same as the above.
